Source/ACE.Entity/AceObjectPropertiesSpell.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using ACE.Common;
+using MySql.Data.MySqlClient;
+namespace ACE.Entity
+{
+    [DbTable("ace_object_properties_spell")]
```

Comment:
That is not a new table - it has always been in ace_world and is populated by the main data load.   It was just never wired up.   All I did was wire it up and load the values.   It is used as part of the magic system for items to store what enchantments are permanently part of the item.   I needed to implement it for appraisal event to work correctly.   It is one of the tables just like ace_properties_int - it was just never wired up before now.